test/gollum/views/test_helpers.rb

```diff
@@ -0,0 +1,137 @@
+require_relative "../../helper"
+require_relative "../../../lib/gollum/views/helpers"
+
+require "minitest/autorun"
+require "minitest/spec"
```

Comment:
`minitext/autorun` and `minitest/spec` are not used in the rest of Gollum's test suite. So as you can see, the tests here are very different than the existing test suite.

I am open to changing these tests to better match the existing suite. However, I do feel that spec-style tests are easier to read and write. I'm curious to hear your thoughts on them, and whether it would be worth refactoring some other tests to use the `spec` DSL.